### ISSUE TITLE
Can't read obj_info header written by CloudCompare

### ISSUE THREAD

**Issue by rreusser** [nmwsharp/happly#12]
Thanks for the great parser! We were so happy to find it. ❤️😄🌈🎉

One issue we ran into: CloudCompare writes a `obj_info` header, e.g.

```ply
ply
format ascii 1.0
comment Created by CloudCompare v2.10.2 (Zephyrus)
comment Created 8/1/19 9:04 AM
obj_info Generated by CloudCompare!
element vertex 4308
property float x
property float y
property float z
element face 8612
property list uchar int vertex_indices
end_header
74.5034 161.291 168.961 
77.047 149.808 164.128 
73.8367 161.097 169.151 
73.1975 160.869 169.417 
...
```

I can ping someone to track down the exact line of code that's failing, but it seems pretty cut and dry that you simply have to remove the `obj_info` line to avoid getting an error about an invalid header. Seems like just a simple obscure PLY header field that's not handled.

We'll probably have to fix this, but at the very least wanted to get it reported. Thanks again!

See also: https://github.com/PointCloudLibrary/pcl/issues/2763#issue-397094140

**Comment by rreusser** [comment]
For a start, maybe just detect it and skip it? https://github.com/nmwsharp/happly/blob/826c5ab0530fd2ce0e608f56eab8ffca4f9da7c6/happly.h#L1485-L1491 Or store as a comment? (Or create a proper data structure for obj_info, but maybe that's overkill for how common/meaningful it is?)

**Comment by nmwsharp** [comment]
Glad hapPLY is useful for you!

Yeah, `obj_info` seems to be common enough that we should definitely accept files which include it...

Easy solution would be to skip such lines, but it seems a nonzero number of users pack important info in to it (see e.g. [here](https://www.cs.jhu.edu/~misha/Code/PoissonMesh/TextureStitcher/range-grid.ply.html)), so it's probably worthwhile to preserve. I'm thinking just dump all `obj_info` lines in to a `std::vector<std::string>>`.

It looks like the recent pull db040e7099088f04dee0a181841194234698144c already added something to handle this, so I'll work a fix in with that soon.

**Comment by nmwsharp** [comment]
This should be fixed as of f196c50. `obj_info` lines are now treated like comments-- they can be both read and written to. Please reopen if anything doesn't seem right!

**Comment by rreusser** [comment]
Thanks so much, @nmwsharp! Your quick response and fix are really, really appreciated! 🎉
